perfkitbenchmarker/linux_packages/aerospike_server.py

```diff
@@ -19,13 +19,13 @@
 
 from perfkitbenchmarker import data
 from perfkitbenchmarker import flags
-from perfkitbenchmarker import vm_util
+from perfkitbenchmarker.linux_packages import INSTALL_DIR
 
 FLAGS = flags.FLAGS
 
 GIT_REPO = 'https://github.com/aerospike/aerospike-server.git'
 GIT_TAG = '3.7.5'
-AEROSPIKE_DIR = '%s/aerospike-server' % vm_util.VM_TMP_DIR
+AEROSPIKE_DIR = '%s/aerospike-server' % INSTALL_DIR
```

Comment:
I appreciate making the change look like the previous item but can we use the same `posixpath.join(x,y)` as used elsewhere?